packages/react/src/Logo/LogoAmsterdam.tsx

```diff
@@ -0,0 +1,13 @@
+import type { SVGProps } from 'react'
+
+const LogoAmsterdam = (props: SVGProps<SVGSVGElement>) => (
+  <svg className="amsterdam-logo__image" role="img" viewBox="0 0 115 40" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
I haven’t researched this.

The [W3C WAI](https://www.w3.org/WAI/search/?q=logo) does add a role and a label:

```
<a lang="en" class="home w3c" href="https://www.w3.org/">
      <svg role="img" aria-label="W3C" viewBox="0 0 68 34" xmlns="http://www.w3.org/2000/svg">
```

Reply:
NVDA reads 'clickable link graphic w3c' there, which is not great imo. [gov.uk](https://www.gov.uk/) hides their logo for screenreaders, NVDA reads 'clickable link Go to the GOV.UK homepage' there. We could also get rid of all these 3 attributes and allow the user to set them themselves, for extra flexibility. This is currently not possible though.

Reply:
This is not really helping guys.. Vincents option: screen reader reads 'Image: "Gemeente Amsterdam"', Arams option doesn't read anything. I would argue that the accessible option is that if you have a link around the image (which should always be the case) you do want to have it read something. [The article](https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/) explains that you can also use `aria-labelledby` to explain the image is labelled by a visually hidden site title in the link for example. A `<title>` tag in the svg bubbles up to the svg (with role image) but this would make the link less accessible and you do not want the site title twice.

Reply:
Hah I understand. I should have been clearer

1. The code snippet doesn’t reflect my opinion, just an example of a site that probably knows what they’re doing on this subject.
2. My implicit question was ‘Can you please research this and implement your best option’.

I would expect a screen reader user to want to hear ‘Homepage, link’ or ‘Link, homepage’ when focusing the link to the logo. Can we make that happen?

Reply:
Yes, it's up to the developer to wrap the logo in a link. We've decided the logo should just be the image.